File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){
    int n;
    do{
        n = get_int("Height: \n");
    }
    while(n < 1 || n > 8);
    {
        for(int i = 0; i < n; i++)
        {
            for(int j = n - i; j > 1; j--){
            printf(" ");
        }
        for (int x = 0; x < i + 1; x++)
        {
            printf("#");
        }
            printf("  ");
            
        for (int y = 0; y < i + 1; y++)
        {
            printf("#");
        }
            printf("\n");

    }


}
}

File: pset6/mario/more/mario.py
from cs50 import get_int


x = 0;

while x < 1 or x > 8:
    x = get_int("Height: ")

for i in range(x):
    print(f"{(x-1-i)*' '}{(i+1)*'#'}  {(i+1)*'#'}")


